core/src/play/java/de/danoeh/antennapod/core/cast/CastUtils.java

```diff
@@ -192,8 +192,10 @@ public static Playable getPlayable(MediaInfo media, boolean searchFeedMedia) {
                 }
             }
             if (result == null) {
-                FeedItem feedItem = DBReader.getFeedItemByUrl(metadata.getString(KEY_FEED_URL),
-                        metadata.getString(KEY_EPISODE_IDENTIFIER));
+                FeedItem feedItem = DBReader.getFeedItemByGuidOrEpisodeUrl(
+                        null,
+                        metadata.getString(KEY_EPISODE_IDENTIFIER)
+                );
```

Comment:
```suggestion
                FeedItem feedItem = DBReader.getFeedItemByGuidOrEpisodeUrl(null,
                        metadata.getString(KEY_EPISODE_IDENTIFIER));
```

Reply:
I prefer stacked list over breaking the code at some arbitrary screen size limit

Reply:
If this was the first ever code in a new app, we could argue about such code style changes. The thing is, all the existing code does not use this style and being consistent within a project is pretty important for readability.

Reply:
OK. I change my code accordingly.

I ask myself how you want to improve a codebase to better code styles if there is only the all or nothing approach?
By applying linebreaks in respect to the type of code (method name, arguments, etc) thus reducing cognitive load to identify what is what improves readability. 
There is also the approach to migrate per change. Meaning you apply a more readable/improved style to new and changed code.